src/cascadia/TerminalSettingsModel/FileUtils.cpp

```diff
@@ -15,14 +15,34 @@
 static constexpr std::string_view Utf8Bom{ "\xEF\xBB\xBF", 3 };
 static constexpr std::wstring_view UnpackagedSettingsFolderName{ L"Microsoft\\Windows Terminal\\" };
 static constexpr std::wstring_view ReleaseSettingsFolder{ L"Packages\\Microsoft.WindowsTerminal_8wekyb3d8bbwe\\LocalState\\" };
+static constexpr std::wstring_view PortableModeMarkerFile{ L".portable" };
+static constexpr std::wstring_view PortableModeSettingsFolder{ L"settings" };
 
 namespace winrt::Microsoft::Terminal::Settings::Model
 {
     // Returns a path like C:\Users\<username>\AppData\Local\Packages\<packagename>\LocalState
     // You can put your settings.json or state.json in this directory.
+    bool IsPortableMode()

```

Comment:
Just so I'm clear here - if we start in portable mode, then I (_a monster_) go delete `.portable` while the Terminal is running, we'll keep using the portable settings and stay in portable mode, till we exit and relaunch the terminal.

Reply:
If you open another Terminal process at this point, it'll still ask the monarch which window it should go into - either way, the original portable process would create the window, just as portable as before. 

That tracks.

Reply:
Yes, if you (a monster) do that... it'll do that.

K, so.

It looks like two different launches from two different portable mode directories will do something ~ ~ weird ~ ~ and glom together (lol!)

Reply:
I want to fix in post, and I have a clever idea for it.